### PR TITLE
refactor: remove json string encoding from span detail

### PR DESCRIPTION
**Pull request by anandtiwary**

### CODE REVIEW

**Comment by aaron-steinfeld** [comment]
What's an example of a before/after body here?

**Review by jake-bassett — APPROVED** [review]

**Comment by anandtiwary** [comment]
Decode the JSON payload --> convert `\"` to `"`

"{\"content\": 2 }" -> {"content": 2}

File: projects/distributed-tracing/src/shared/components/span-detail/call/body/span-detail-call-body.component.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ import { ToggleViewMode } from '@hypertrace/components';
             </ht-message-display>
           </section>
           <section *ngSwitchCase="this.rawLabel">
-            <pre class="pre-body">{{ this.body | json }}</pre>
+            <pre class="pre-body">{{ this.rawBody }}</pre>
```
Comment (aaron-steinfeld) [inline projects/distributed-tracing/src/shared/components/span-detail/call/body/span-detail-call-body.component.ts:33]: Are we not pretty printing this anymore? Did this ever work - I thought the json pipe takes objects.

**Review by aaron-steinfeld — COMMENTED** [review]
> Decode the JSON payload --> convert `\"` to `"`
> 
> `"{\"content\": 2 }" `-> {"content": 2}

I saw the conversion of `\"`, but not the leading or trailing quotes.

File: projects/distributed-tracing/src/shared/components/span-detail/call/body/span-detail-call-body.component.ts (hunk 1 above)
Comment (anandtiwary) [reply projects/distributed-tracing/src/shared/components/span-detail/call/body/span-detail-call-body.component.ts:33]: Yep. Don't think it worked for raw format.

**Review by aaron-steinfeld — APPROVED** [review]

**Pull request merged**